### ISSUE TITLE
Windscribe Broken

### ISSUE THREAD

**Issue by Kastilo** [EnergizedProtection/block#5]
Hello, it seems that when I try to connect to Windscribe on Ubuntu while Energized Protection is enabled it fails. This happens when trying to establish the firewall and after the hosts is cleared the firewall is restored and I am able to connect. Not sure what is going on since this is a brand new issue with the protection

OS: Ubuntu 18.04 LTS
Filter List: Ultimate

**Comment by k4rn8g3** [comment]
@Kastilo hm..
since this is a free vpn..
it might have trackers and ads injecting which are blocked by energized and hence , the vpn doesnt work with energized hosts..

**Comment by Kastilo** [comment]
This issue isn't present on my Android with the same protection settings as my Desktop set up and the same VPN is running, so it's not the VPN it seems

**Comment by k4rn8g3** [comment]
@Kastilo Maybe both versions (like Linux stuff and Android stuff.. How both works, their mechanisms, dependencies etc..)

In short ..

The implementation may be different on Android..

Will have a look and see what's possible from our side..

**Comment by Kastilo** [comment]
Thank you, I manually added the DNS and other information to the hosts and Windscribe works again, but it gets wiped, so hopefully the yourself and the Energized team can figure this out. Thank you for looking into it
